src/devices/Interop/Unix/Libc/Interpo.ioctl.h.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
nit: typo in file name

Reply:
Optional: I'd prefer this file was called `Interop.ioctl.macros.cs` since I think that's what those are